authors/apps/favorite/views.py

```diff
@@ -0,0 +1,111 @@
+from rest_framework.generics import CreateAPIView, DestroyAPIView, ListAPIView
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from rest_framework import status
+
+# local imports
+from .serializers import FavoritedSerializers
+from .renderers import FavoriteJSONRenderer
+from ..articles.models import Article
+from .models import FavouriteArticle
+
+
+class FavoriteView(CreateAPIView, DestroyAPIView):
+    """
+    favorite and unfavorite an article.
+    """
+    permission_classes = (IsAuthenticated,)
+    serializer_class = FavoritedSerializers
+    renderer_classes = (FavoriteJSONRenderer,)
+
+    def post(self, request, slug):
+        """
+        Favorite a specific article
+        :param slug
+        authenticated users can favorite an article
+        """
+
+        res = check_exist(slug)
```

Comment:
Have a more descriptive variable name for the res variable. Its not python standard